packages/motion/src/helpers.js

```diff
@@ -85,7 +85,7 @@ export function getWebpackConfig(state: State, config: Motion$Config, cli: CLI,
         Path.join(Path.normalize(Path.join(__dirname, '..')), 'node_modules'),
         Path.join(Path.normalize(Path.join(__dirname, '..', '..')), 'node_modules')
       ],
-      packageMains: ['webpack', 'browser', 'web', 'browserify', 'jsnext:main', 'main']
+      packageMains: ['webpack', 'browser', 'web', 'browserify', 'main', 'jsnext:main']
```

Comment:
The main reason we're ignoring `jsnext:main` is because we are excluding `node_modules` in the babel loader, `jsx:next` points to a file containing un-babeled sources.

Therefore instead of babelifying everything just to support jsxnext:main, we are only transpiling the app.